server/tserver/src/main/java/org/apache/accumulo/tserver/replication/AccumuloReplicaSystem.java

```diff
@@ -370,11 +368,112 @@ protected Status replicateLogs(ClientContext peerContext, final HostAndPort peer
 
     log.debug("Replication WAL to peer tserver");
     final Set<Integer> tids;
-    final DataInputStream input;
-    Span span = Trace.start("Read WAL header");
-    span.data("file", p.toString());
-    try {
-      input = getWalStream(p);
+    try (final FSDataInputStream fsinput = fs.open(p); final DataInputStream input = getWalStream(p, fsinput)) {
```

Comment:
nit: would prefer to see these split up onto two lines. Not sure what others think :)

Reply:
The formatter in the build process reformats it so they are on the same line.

Reply:
That might be configurable. Would have to investigate the options in newer Eclipse versions. It doesn't matter to me... if it's configurable, I'll defer to others as to what is most readable.